challenges/objects-arrays.js

```diff
@@ -98,14 +242,23 @@ The zoos need a list of all their animal's names (names only, not scientific) co
 */
 
 const lowerCase = [];
-console.log(lowerCase); 
+
+lowerCase = zooAnimals.map(animalData => {
+  return animalData.animal_name.toLowerCase();
+});
+console.log(lowerCase);
 
 /* Request 3: .filter() 
 
 The zoos are concenred about animals with a lower population count. Find out which animals have a population less than 5.
 
 */
 const lowerPopulation = [];
```

Comment:
remember to comment out this line so your code works
const lowerPopulation = [];